src/PrintOrder/utils.js

```diff
@@ -64,3 +64,30 @@ export const getOrderPrintData = async (ky, order = {}) => {
     shipToRecord: addressMap[order.shipTo],
   });
 };
+
+export const getCurrencyRate = async (ky, from, to) => {
+  const currencyRate = await ky.get(`finance/exchange-rate?from=${from}&to=${to}`).json().catch(() => ({}));
+
+  return currencyRate?.exchangeRate;
+};
+
+export const getPOLineTotalEstimatedPrice = async ({ ky, poLine = {}, systemCurrency }) => {
+  const {
+    currency,
+    exchangeRate,
+    poLineEstimatedPrice,
+    quantityElectronic = 0,
+    quantityPhysical = 0,
+  } = poLine;
+
+  let currentExchangeRate = exchangeRate;
+
+  if (currency !== systemCurrency && !exchangeRate) {
+    currentExchangeRate = await getCurrencyRate(ky, currency, systemCurrency);
+  }
+
+  return {
+    totalItems: quantityPhysical + quantityElectronic,
```

Comment:
how is it connected to price?

Reply:
Not sure. I think you worked on this implementation. I just moved the line to utils from here https://github.com/folio-org/ui-orders/pull/1560/files/001714803ece1b7662f328a5721c8b0b106fc734#diff-5f2ed363915b466f2a51ba819555fc6565621677eae5c8a15824e5f9c05d9b97L17-L22

Reply:
but you moved to price specific util, how quantity is connected to price?